openpower/package/hostboot/p9Patches/hostboot-0002-remove-attr-override-compile.patch

```diff
@@ -1,13 +1,29 @@
-From d71055295a21e7fe9105bdfe0085006462ab568c Mon Sep 17 00:00:00 2001
+From 53bd51536cccbef8d080a92bf4a2b16ca549af3c Mon Sep 17 00:00:00 2001
 From: Matt Ploetz <maploetz@us.ibm.com>
-Date: Tue, 9 Aug 2016 10:08:15 -0500
-Subject: [PATCH] Remove attribute override compile
+Date: Tue, 6 Sep 2016 14:08:08 -0500
+Subject: [PATCH] Remove the building and distributuion of attribute override
```

Comment:
Yes, this will be resolved with C++11 support and yes, it is still in the queue to be worked on.